src/third/redux-persist/createPersistor.js

```diff
@@ -116,9 +122,13 @@ export default function createPersistor (store, config) {
   }
 }
 
-function warnIfSetError (storageKey) {
-  return function setError (err) {
-    if (err && process.env.NODE_ENV !== 'production') { console.warn('Error storing data for storage key:', storageKey, err) }
+function warnIfSetErrors (storageKeys) {
+  return function setErrors (errors) {
+    if (errors && errors.length > 0 && process.env.NODE_ENV !== 'production') {
+      errors.forEach(err => {
+        console.warn('Error storing data for one of these storage keys:', storageKeys, err)
+      })
```

Comment:
Can we make this error function better?

How about we have it print out storageKeys once at the top, then each error individually after that?